occurrences/models.py

```diff
@@ -289,6 +321,57 @@ def __str__(self):
         return f"{self.id} {self.name}"
 
 
+class EnrolmentQuerySet(models.QuerySet):
+    def send_enrolment_summary_report_to_providers(self, days=1):
+        reports = {}
+        # New auto accepted enrolments or pending enrolments during the last `days`
```

Comment:
I think this comment is a bit misleading as `days` do not affect the pending enrolments.